bbn-test-5/babylon-validators/README.md

```diff
@@ -0,0 +1,187 @@
+# Babylon Validator Setup
+
+## Table of Contents
+
+1. [Prerequisites](#prerequisites)
+2. [Setup](#setup) \
+   2.1. [Setup Node and Configure](#setup-node-and-configure) \
+   2.2. [Build node from `x` tag](#build-node-from-x-tag) \
+   2.3. [Initialize the chain](#initialize-the-chain) \
+   2.4. [Update the configuration](#update-the-configuration) \
+   2.5. [Retrieve the genesis file](#retrieve-the-genesis-file) \
+   2.6. [Create a Keyring](#create-a-keyring) \
+   2.7. [Get Funds](#get-funds) \
+   2.8. [What is BLS Voting](#what-is-bls-voting) \
+   2.9. [Create BLS Key](#create-bls-key)
+3. [Run Node](#run-node)
+4. [Create Validator](#create-validator)
+5. [Verify your Validator](#verify-your-validator)
+
+## Prerequisites
+
+This system spec has been tested by validators and found to be comfortable:
+
+- Quad Core or larger AMD or Intel (amd64) CPU
+- 32GB RAM
+- 1TB NVMe Storage
+- 100MBps bidirectional internet connection
+
+You can run Babylon on lower-spec hardware for each component, but you may find that it is not highly performant or prone to crashing.
+## Setup
```

Comment:
I don't think we need this section. I recommend we have the following steps:

1. Install Babylon binary (download go, download babylon codebase, build/install).
2. Setup your home directory and configure it.
3. Setup the required keys for operating a validator (CometBFT key and BLS key) -- should contain an explanation of what these keys are and why they are needed. 
4. Sync your node (brief explanation on where to find nodes to connect to or a snapshot -- no need to give much tech details on that). 
5. Receive the necessary funds for creating your validator (explain why funds are needed).
6. Create validator (with brief explanation of the epoching mechanism).